# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../iproute2/0003-include-poll.h.patch
# Copyright (C) 2013 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

include poll.h instead of sys/poll.h

http://pubs.opengroup.org/onlinepubs/009695399/basedefs/poll.h.html

NAME

  poll.h - definitions for the poll() function

SYNOPSIS

  #include <poll.h> 
 
--- ./misc/nstat.c.orig	2013-08-29 02:05:41.157849476 +0200
+++ ./misc/nstat.c	2013-08-29 02:06:10.838522289 +0200
@@ -21,7 +21,7 @@
 #include <sys/file.h>
 #include <sys/socket.h>
 #include <sys/un.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
 #include <signal.h>
--- ./misc/ifstat.c.orig	2013-08-29 02:07:38.468597613 +0200
+++ ./misc/ifstat.c	2013-08-29 02:07:51.040899211 +0200
@@ -21,7 +21,7 @@
 #include <sys/file.h>
 #include <sys/socket.h>
 #include <sys/un.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
 #include <signal.h>
--- ./misc/rtacct.c.orig	2013-08-29 02:08:26.333684280 +0200
+++ ./misc/rtacct.c	2013-08-29 02:08:40.030008554 +0200
@@ -22,7 +22,7 @@
 #include <sys/file.h>
 #include <sys/socket.h>
 #include <sys/un.h>
-#include <sys/poll.h>
+#include <poll.h>
 #include <sys/wait.h>
 #include <sys/stat.h>
 #include <sys/mman.h>
